site/styles/_meta-sub-item.scss

```diff
@@ -110,3 +111,19 @@
 .s72-playback-progress-bar {
   background-color: var(--secondary);
 }
+
+s72-show-hide-switcher {
+  display: block;
+  margin-bottom: 15px;
+  padding-bottom: 15px;
+  transition: height 0.5s ease;
+
+  @include media-breakpoint-up(sm) {
+    height: auto !important;
```

Comment:
It looks like this line has no effect because the component is setting the correct height anyway.  Can it be removed?

Reply:
Oh wow it is indeed, pretty cool it still calculates above sm.. I mean, totally meant to do that.

- removed height and breakpoint up